packages/augur-ui/src/modules/modal/modal.styles.less

```diff
@@ -2944,7 +2944,6 @@ div.ButtonsRow:last-of-type {
         margin-bottom: @size-16;
       }
 
-      > div:nth-of-type(2),
```

Comment:
Can you check that spacing? I think there's a margin-top there or something. The first element to the right is miss-aligned.
![Screenshot at Feb 06 15-49-50](https://user-images.githubusercontent.com/10351013/73969863-e9d07c80-48fa-11ea-9273-8adccfaeb721.png)